### PR TITLE
Added custom adr algorithm for slow data-rate correction

### PR DESCRIPTION
**Pull request by MStenkaer**
Added custom adr algorithm with less aggressive dr correction, used for areas with varying snr.
Added description on how to create and build new adr plugins.

Needed for resolving https://github.com/OS2iot/OS2IoT-frontend/issues/103

### CODE REVIEW

**Comment by ramogens** [comment]
Thank you very much for making this PR, @MStenkaer. We have started the process of getting a review, and as of now we'll be able to do it in October or November.

File: README.md
Comment (AramAlsabti) [inline README.md:85]: Is the backtick at the end of the line a mistake?
Same comment to the line below

File: README.md
Comment (AramAlsabti) [inline README.md:87]: Aren't you missing the name of the module after `init`?

File: README.md
Comment (AramAlsabti) [inline README.md:80]: Thank you for the pre-built module! For security reasons, we will rebuild it with what's given and update it after the PR.

**Review by AramAlsabti — CHANGES_REQUESTED** [review]
Pretty good! Only minor suggestions to the readme

File: configuration/chirpstack-network-server/adr-modules/sensade-adr-mod/go.mod
Comment (MStenkaer) [reply configuration/chirpstack-network-server/adr-modules/sensade-adr-mod/go.mod:6]: You are right that it seems wrong. However, this is autogenerated during 'go build' and should include '/v3'

File: README.md
Comment (MStenkaer) [reply README.md]: You are right, the backticks has been remove on both lines

File: README.md
Comment (MStenkaer) [reply README.md]: You should be able to do it without, but to make it more general I have added it.

File: README.md
Comment (MStenkaer) [reply README.md:80]: You are right about that. 
Do you want me to exclude the one that I build for the PR?

Comment (AramAlsabti) [reply README.md:80]: Good point. You're welcome to do that.

File: README.md
Comment (AramAlsabti) [reply README.md]: You're right. It error'd out when I attempted it. It might have had something to do with path not being registered in Go

**Comment by AramAlsabti** [comment]
Once you've removed the pre-built moduled, you're welcome to merge the PR.

**Review by AramAlsabti — APPROVED** [review]
Once you've removed the pre-built moduled, you're welcome to merge the PR.

**Comment by MStenkaer** [comment]
> Once you've removed the pre-built moduled, you're welcome to merge the PR.

I have done that. I do not have write access to the repository, so I cannot merge the PR,

**Pull request merged**
